docs/docusaurus.config.ts

```diff
@@ -31,6 +33,42 @@ const config: Config = {
     locales: ["en"],
   },
 
+  plugins: [
+    [
+      "docusaurus-plugin-llms-builder",
+      {
+        version: packageJson.version,
+        llmConfigs: [
+          {
+            title: "Gobley",
+            description: "Embed Rust into your Kotlin Multiplatform project",
+            summary: `
+A set of libraries and tools that help you mix Rust and Kotlin
+using [UniFFI](https://github.com/mozilla/uniffi-rs). This project was forked
+from [UniFFI Kotlin Multiplatform bindings](https://gitlab.com/trixnity/uniffi-kotlin-multiplatform-bindings).
+Since the original project is no longer maintained, active development now continues here.
+
+Currently, Android, Kotlin/JVM, and Kotlin/Native are supported. WASM is not supported yet.
+
+- UniFFI Bindings Generation for Kotlin Multiplatform (Android, JVM, Kotlin/Native)
+- KotlinX Serialization Support
+- Automatic building and linking of Rust libraries to Kotlin projects
+            `,
```

Comment:
I think `summary: fs.readFileSync(path.join(__dirname, "..", "README.md"), { encoding: "utf-8" })` is a valid option here, though it'll include the badges and extra headings. What do you think?